Organizer-Timeline/1-Month-Before/Workshops.md

```diff
@@ -0,0 +1,32 @@
+# Finding Great Workshop Content at O(1) Speed
+
+As a hackathon organizer, you have to worry about getting enough food, figuring out the budget, setting the schedule, contacting sponsors, etc. Then you realize your hackathon doesn't have any workshops on its schedule, Or you don't know what workshops can appeal to your hackers, all with different experience levels. You want your workshop to be fun and educational. But your team is already busy managing everything else, and the last thing you need is to worry about finding time to prepare slides, dig through API documentation, and setup a development environment.
+
+There's no need to fear, [Localhost is here](https://localhost.mlh.io)! Localhost provides workshops that you can use right out of the box. 
+
+## What MLH Localhost provides
+
+We provide you a slide deck that is easy to follow, all of the documentation, lots of swag, and any hardware that you might need. All given to you for free. You don't need any prior experience with the tech that you are teaching. Localhosts are designed for beginners to teach other beginners. Just practice the slides before the workshop, run the workshop, and give out the swag after the workshop. Profit.  
```

Comment:
We provide you with a step-by-step presentation that is easy for people of all skill levels to follow, all of the documentation, lots of swag, and loan you any hardware that you might need – all for free! 

Localhost workshops are designed .  . . 

The word "profit" at the end is hilarious but let's take it out, because we don't want participants to think they can charge for the workshops.